include/tgfx/core/Image.h

```diff
@@ -332,6 +332,14 @@ class Image {
   virtual std::shared_ptr<Image> onMakeScaled(int newWidth, int newHeight,
                                               const SamplingOptions& sampling) const;
 
+  /**
+   * Returns given scale if the subclass's lockTextureProxy method supports direct downscaling; otherwise,
+   * returns 1.0f.
+   */
+  virtual float getRasterizedScale(float) const {
```

Comment:
把参数补上 ，drawScale。

Reply:
Returns the scale factor for rasterized caching based on the given draw scale if the subclass's lockTextureProxy method supports direct downscaling; otherwise, returns 1.0f.

Reply:
Returns the scale factor for rasterized cache based on the given draw scale if the subclass's lockTextureProxy method supports direct downscaling; otherwise, returns 1.0f.